File: javascript/calculadoras/calc.py

#primeiro numero

num1 = float(input("Escolha um numero\n"))

#qual a operação

operacao = input("escolha entre (+, -, /, *)")

#caso tenha escrevido algo errado

while operacao not in ["+", "-", "/", "*"]:
  operacao = input("escolha entre (+, -, /, *)")
  
#segundo numero

num2 = float(input("escolha outro numero\n"))

#Encontra o restado

if (operacao in "+"):
  resposta = num1 + num2
elif (operacao in "-"):
  resposta = num1 - num2
elif (operacao in "*"):
  resposta = num1 * num2
else:
  resposta = num1 / num2
  
#mostra o resultado ao usuario

print(resposta)

File: javascript/calculadoras/calc.js
const $num1 = parseFloat(prompt("Escolha um número:\n"));
let $operacao = (prompt("Escolha entre (+, -, *, /)"));
while (!["+", "-", "*", "/"].includes($operacao)) {
  $operacao = (prompt("Escolha entre (+, -, *, /)"));
}
const $num2 = parseFloat(prompt("Escolha outro número:\n"));
let $resultado = (0);
if (["+"].includes($operacao)) {
  $resultado = ($num1 + $num2)
} else if (["-"].includes($operacao)) {
  $resultado = ($num1 - $num2)
} else if (["*"].includes($operacao)) {
  $resultado = ($num1 * $num2)
} else {
  $resultado = ($num1 / $num2)
}
console.log(`O resultado de ${$num1} ${$operacao} ${$num2} é ${$resultado}`)